app/services/foreman_rh_cloud/cloud_request_forwarder.rb

```diff
@@ -34,7 +34,13 @@ def prepare_request_opts(original_request, forward_payload, forward_params, cert
         ),
       }
       requested_url = original_request.original_fullpath.end_with?('/') ? original_request.path + '/' : original_request.path
-      base_params.merge(path_params(requested_url, certs))
+      path_params = path_params(requested_url, certs)
```

Comment:
Can you call the `path_params` variable some other name to differentiate it from the method in the same object

Reply:
Done